src/rgw/rgw_auth.cc

```diff
@@ -196,7 +196,7 @@ strategy_handle_denied(rgw::auth::Engine::result_t&& engine_result,
       return std::make_pair(true, std::move(engine_result));
 
     case Control::FALLBACK:
-      return std::make_pair(true, std::move(strategy_result));
+      return std::make_pair(true, std::move(engine_result));
```

Comment:
shouldn't we stick with `strategy_result` for `strategy_handle_denied()`?

Reply:
Yes, my apologies again, I missed reverting this change.